app/models/communication.rb

```diff
@@ -60,6 +61,20 @@ def self.create_with_twilio_details!(appointment:, twilio_sid:, twilio_msg_statu
     end
   end
 
+  def self.create_with_imo_details!(appointment:, notification:, result:)
```

Comment:
What are we doing with the `result`?

Reply:
It'll go into `TwilioSmsDeliveryDetail` in one of the upcoming tickets. I'm starting to think I should've made that a bit clearer.

Reply:
Or we can add things like `result` into that subsequent PR only, rather than into this one. That woudl eliminate the need to explain the sequence of events too deeply everywhere.